egs/swbd/s5c/local/nnet3/run_lstm.sh

```diff
@@ -19,36 +31,43 @@ hidden_dim=1024
 recurrent_projection_dim=256
 non_recurrent_projection_dim=256
 chunk_width=20
-chunk_left_context=20
-clipping_threshold=5.0
+chunk_left_context=40
```

Comment:
%WER 44.1  VD-LSTM with chunk-width 30 and chunk-left-context 30 (Total training time is 17:04:14)
%WER 43.8  VD-LSTM with chunk-width 20 and chunk-left-context 40 (Total training time is 18:43:35)

Not much benefit in terms of speed and slightly worse results. 

I ensured that the frames-per-iter is the same in both cases by using 
13334 chunks per iteration x 30 chunk-width
20000 chunks per iteration x 20 chunk-width

VD-LSTM : Variable delay LSTM